### PR TITLE
Iss #13 sql

### PR DESCRIPTION
**Pull request by stephenholleran**
A SQL file which contains CREATE TABLE statements to define a database schema which fits the data model standard.

Edit: based on this issue #13

### CODE REVIEW

File: tools/iea43_wra_data_model_postgresql.sql [hunk 1]
```diff
@@ -0,0 +1,378 @@
+-- These PostgreSQL statements are a relational database representation of the
+-- IEA Wind Task 43 Wind Energy Digitalization Standardized Data Model
+
+-- Running these SQL statements in a PostgreSQL database will create the database
+-- schema and insert named reference values (enums).
+
+-- The SQL standard has tried to be used as much as possible.
```
Comment (flrs-edf) [inline tools/iea43_wra_data_model_postgresql.sql:7]: @stephenholleran it is not clear to me what this means.

**Review by flrs-edf — COMMENTED** [review]
I was able to create a database from the iea43_wra_data_model_demo_data.sql file and insert the demo data into the database with the iea43_wra_data_model_postgresql.sql file.

From my perspective, the SQL statements look fine.

If our goals are to have this database be very scalable and well-performing, I would suggest to bring in a professional data(base) engineer to review the layout in light of these goals, as this is currently beyond my skill set.

File: tools/iea43_wra_data_model_postgresql.sql (hunk 1 above)
Comment (stephenholleran) [reply tools/iea43_wra_data_model_postgresql.sql:7]: > I would suggest to bring in a professional data(base) engineer to review the layout in light of these goals, as this is currently beyond my skill set.

Do you guys have a database person in house that could take a look?

> it is not clear to me what this means: `The SQL standard has tried to be used as much as possible`.

The SQL Standard ([this ](https://www.oreilly.com/library/view/sql-in-a/1565927443/ch01s03.html) is the best link I could find) standardizes the SQL language so the same statement can be used in different SQL databases. However some databases don't fully follow the standard and have their own versions. I have tried to use data types and keywords that are not Postgresql specific and so the statements could be used in other RDBMS like MySQL. I wasn't able to fully do it as it fails with MySQL but hopefully there are less modifications to be made.

Comment (stephenholleran) [reply tools/iea43_wra_data_model_postgresql.sql:7]: Hey @flrs-edf, 

Any chance you have time to give the ok for this if you are happy with it?

I have added the two changes relating to `instrument_poi_height_mm` and `lattice_has_horizontal_member`.

Cheers!

**Pull request merged**